src/mon/OSDMonitor.cc

```diff
@@ -796,14 +796,20 @@ class OSDUtilizationDumper : public CrushTreeDumper::Dumper<F> {
 
   bool get_osd_utilization(int id, int64_t* kb, int64_t* kb_used,
 			   int64_t* kb_avail) const {
+    // we always return true for OSDs as the specific OSDs can be out,
```

Comment:
could you be more specific in the commit message, "chaos" is sort of vague, imho.